spec/requests/static_pages_spec.rb

```diff
@@ -21,4 +21,20 @@
       expect(response.body.include?('ADMIN')).to eq(true)
     end
   end
+
+  describe 'Pagination' do
+    it 'body must match' do
```

Comment:
make this block name more clear to understand. for example it 'returns first page if -1 is requested'